### ISSUE TITLE
Operator containsText matchs different results with and without fulltext index

### ISSUE THREAD

**Issue by studiodev** [orientechnologies/orientdb#2036]
Hello,

I've just noticed an unusual behavior with `containsText` operator. The result is not the same with or without fulltext index.

Without index, containsText matches with **partial** words, and with index, only **exact** words.

    CREATE CLASS Car
    CREATE PROPERTY Car.name STRING
    INSERT INTO Car SET name = 'JAMIE'
    INSERT INTO Car SET name = 'JAMES'
    INSERT INTO Car SET name = 'JALES'

    SELECT FROM Car WHERE name containsText 'JAM' // 2 results
    CREATE INDEX Car.name FULLTEXT
    SELECT FROM Car WHERE name containsText 'JAM' // 0 result

I suppose exact matching is what is expected. Partial matching is really useful too ;) 

Tested on `develop`.

**Comment by zifnab87** [comment]
Hello @lvca is there any update with this enhancement? Thanks :)

**Comment by lvca** [comment]
I'm forwarding this to Enrico that worked on Lucene plugin.

**Comment by maggiolo00** [comment]
@studiodev 
which version of OrientDB?
i tried with the latest snapshot and works even with index

**Comment by lvca** [comment]
I'm closing this issue. In case you're able to reproduce it again against last 2.1, please comment/reopen this issue. Thanks.

**Comment by cjayawickrema** [comment]
@lvca,
We experienced this same issue on orientdb 2.1.
Reading through this thread we tried a string with minimum 3 characters and it worked. However, the moment this string gets a space it doesnt work again. Is there a way to handle this?

e.g.
This worked
SELECT FROM Employee WHERE fullName containstext 'Tony'

This didn't
SELECT FROM Employee WHERE fullName containstext 'Tony Star'

I have a FULLTEXT HASHED INDEX for this fullName field.

Appreciate your help.
